components/automate-chef-io/content/docs/configuration.md

```diff
@@ -251,10 +251,10 @@ Chef Automate comes with a default LDAP configuration for Microsoft Active Direc
 The MSAD configuration is intended for getting started with minimal configuration for standard MSAD systems.
 Default values in this MSAD configuration can be overridden.
 
-{{< warning >}}
+{{% warning %}}
```

Comment:
Is that just changing a style or is it changing something incorrect to be correct? Is this Hugo syntax, or something else...?

Reply:
The < variant takes its content verbatim and doesn't render it. This content has markup, so it needs rendering. It's Hugo, the word for it is "shortcode".

Reply:
https://github.com/chef/automate/commit/f9b3a450294930910b24c6da042e1e20fa9794fd has had that info, too :wink: